calcom/cal.com
# fix: Messaging app cateogry not being respected

## What does this PR do?



This app fixes bugs caused by the introduction of the "Messaging" app category.
    - On the installed apps page, messaging apps were not being properly rendered in the right category
    - Messaging apps were not being displayed as an option in event type locations

In the following examples Telegram is the messaging app.

## Before
![CleanShot 2023-08-08 at 23 17 15](https://github.com/calcom/cal.com/assets/65426560/c7bafc6e-1262-424f-99ee-cf3f1887249a)

![CleanShot 2023-08-08 at 23 16 43](https://github.com/calcom/cal.com/assets/65426560/691260a1-29e8-46b8-bd27-16d2f49aecbe)

![CleanShot 2023-08-08 at 23 16 31](https://github.com/calcom/cal.com/assets/65426560/5f6a4441-c2de-482e-a1fd-ebbb7d41e545)


## After
![CleanShot 2023-08-08 at 23 14 20](https://github.com/calcom/cal.com/assets/65426560/077cfa56-b480-4d4a-806d-659388e46999)

![CleanShot 2023-08-08 at 23 15 21](https://github.com/calcom/cal.com/assets/65426560/815ae223-d09b-4faf-b178-e34d94d9a18c)

![CleanShot 2023-08-08 at 23 15 36](https://github.com/calcom/cal.com/assets/65426560/44f478f7-bf9e-4104-ad74-6594aa6da803)




## Type of change




## How should this be tested?



- Install a messaging app ex. Telegram
- On the installed app page it should show up under "Messaging" and not "Conferencing"
- In an event type, it should appear as an option

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
